Documentation: Deploying SREs requires Owner role (#2433)
alan-turing-institute/data-safe-haven: docs/source/deployment/deploy_sre.md

::::

## Requirements

> - An Azure subscription where you will deploy your infrastructure.
>     - An account with at least [Owner](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles/privileged) permissions on this subscription.

Reviewer: ```suggestion
- An Azure subscription where you will deploy your infrastructure.
- An account with at least [Owner](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles/privileged) role over the scope of the subscription.
```

A bit of a minor language rework.

I think this makes more sense as a two item list, although I'm open to being told otherwise 😆
